example/go.mod

```diff
@@ -1,11 +1,14 @@
 module github.com/google/go-github/v69/example
 
-go 1.22.0
+go 1.23.1
+
+toolchain go1.23.5
```

Comment:
These changes will have to wait until be bump the Go toolchain for the entire repo.

Reply:
OK, please delete lines 4 and 5 and merge in the latest changes from the `master` branch which should make line 3 say `go 1.23.0` and then we should be good to go.